test/test/util/ui_utils.js

```diff
@@ -169,7 +169,9 @@ shaka.test.UiUtils = class {
     const video = /** @type {!HTMLVideoElement} */(document.createElement(
         'video'));
 
-    video.muted = true;
+    if (!shaka.util.Platform.isTizen()) {
```

Comment:
This needs a comment.  It looks arbitrary without one.  Here's a guess:

```js
    // Tizen has issues with audio-only playbacks on muted video elements.
    // Don't mute Tizen.
```

If that's wrong, please add a comment that is accurate.

Reply:
Added!